docs/source/reference/numpysupported.rst

```diff
@@ -498,6 +498,7 @@ The following top-level functions are supported:
   must be an integer)
 * :func:`numpy.roots`
 * :func:`numpy.rot90` (only the 2 first arguments)
+* :func:`numpy.logspace` (only the 4-argument form)
```

Comment:
```suggestion
* :func:`numpy.logspace` (only the first 4 arguments)
```
given two of the args are kwargs, think it'd be ok to do `logspace(1, 10)` which is a 2-arg form.

Reply:
I am not sure about the method for counting argument number of a function with optional argument, so i just follow the `rot90`, which said  `only the 2 first arguments` when it have a necessary argument `arr` and an optional argument `k`.